draft-ietf-masque-connect-udp.md

```diff
@@ -45,119 +55,227 @@ document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
 In this document, we use the term "proxy" to refer to the HTTP server that
-opens the UDP socket and responds to the CONNECT-UDP request. If there are
-HTTP intermediaries (as defined in Section 2.3 of {{RFC7230}}) between the
+opens the UDP socket and responds to the UDP proxying request. If there are
+HTTP intermediaries (as defined in {{Section 3.7 of SEMANTICS}}) between the
 client and the proxy, those are referred to as "intermediaries" in this
 document.
 
+Note that, when the HTTP version in use does not support multiplexing streams
+(such as HTTP/1.1), any reference to "stream" in this document represents the
+entire connection.
+
+
+# Configuration of Clients {#client-config}
+
+Clients are configured to use UDP Proxying over HTTP via an URI Template
+{{!TEMPLATE=RFC6570}}. The URI template MUST contain exactly two variables:
+"target_host" and "target_port". Examples are shown below:
+
+~~~
+https://masque.example.org/{target_host}/{target_port}/
+https://proxy.example.org:4443/masque?h={target_host}&p={target_port}
+https://proxy.example.org:4443/masque{?target_host,target_port}
+~~~
+{: #fig-template-examples title="URI Template Examples"}
+
+Since the original HTTP CONNECT method allowed conveying the target host and
+port but not the scheme, proxy authority, path, nor query, there exist proxy
+configuration interfaces that only allow the user to configure the proxy host
+and the proxy port. Client implementations of this specification that are
+constrained by such limitations MUST use the default template which is defined
+as: "https://{proxy_host}:{proxy_port}/{target_host}/{target_port}/" where
```

Comment:
A new issue since already merged, but would this be better as a .well-known URI for the default template?

Reply:
Opened #56